anaconda_project/project_file.py

```diff
@@ -119,7 +119,7 @@ class ProjectFile(YamlFile):
 # Use `anaconda-project add-env-spec` to add environment specs.
 #
 env_specs: {}
-'''
+'''.replace('<pkg_key>', YamlFile.pkg_key)
```

Comment:
There are places (like line 412 in this file) where the original string contains empty dictionaries `{}`.  If we use `.format()` then we need to also convert all of these to `{{}}` etc.  I started using `.format()` but then felt it was more explicit if we use a replacement of something that would be unlikely to appear in a `.yaml` file so that escaping of braces is not needed.